### DISCUSSION TITLE
I3S mesh pyramid rendering quality

### DISCUSSION THREAD

**Discussion by bortomar-intellmaps** [visgl/deck.gl#8083] [Q&A]
Hi, we have problem with level of details of I3S mesh pyramid visualisation in deck.gl.  When we visualize same data in Agisoft Viewer, textures look much better. Are there any options that significantly affect the quality of the rendered data in Tile3D layer or i3s loader?

## deck.gl
![image](https://github.com/visgl/deck.gl/assets/135999694/5848e51b-b06b-42f1-a854-63341c5bf64a)
![image](https://github.com/visgl/deck.gl/assets/135999694/d74e0f9c-5421-49b3-8cb9-ffbde55f8550)

## Agisoft Viewer
![image (3)](https://github.com/visgl/deck.gl/assets/135999694/e8579455-a2ce-4eb6-b2be-143aae53c4c7)

**Comment by ibgreen** [comment]
There was a recent patch that fixes a situation that stops the loading of finer resolution tiles. Check your `@loaders.gl/...` versions, I believe the patch went into `3.4.11` hopefully that will improve the situation.

**Reply by bortomar-intellmaps** [reply]
Thank you for quick reply. I upgraded @loader.gl from 3.4.6 to 3.4.11 but unfortunately I don't see any difference.

**Comment by felixpalmer** [comment]
@bortomar-intellmaps take a look at the debugging actions I took here with a similar bug: https://github.com/visgl/deck.gl/issues/7955#issuecomment-1677273635. By using the I3S explorer you can verify what is being loaded and try and figure out what the issue is. 

To move this forward, could you share a minimal live demo with your dataset that demonstrates the issue? It's not possible to debug just from a pair of screenshots
